_spinnaker/using_dinghy.md

```diff
@@ -861,3 +861,97 @@ You have probably configured global logging levels with `spinnaker-local.yml`. T
     Logging:
       Level: INFO
     ```
+
+
+# Webhook secret validation
+
+If you want to add a layer of security or restrict which repositories dinghy will process you can do it with this feature.
+
+Enabling webhook secret validation you ensure that your service provider is the only one that can trigger your pipelines, not an imposter.
+
+At this moment only **github** version control provider is supported.
+
+
+## Enabling/Disabling Webhook secrets
+When you enable Webhook secrets validation, **ALL** webhooks for that provider will be validated to have a secret, more behavior details can be found in Configuring Webhook secrets.
+
+* **Operator**
+
+    ```yaml
+    I should ask for operator?
```

Comment:
You can run the hal command and look in `~/.hal/config` under `armory.dinghy`. Then just copy the contents to operator's `SpinnakerService` manifest under `spec.spinnakerConfig.config.armory.dinghy`.
By the way, you may need to also update https://docs.armory.io/spinnaker/armory_halyard/#hal-armory-dinghy-edit

Reply:
Thanks for the observation!! I just added the commands in halyard section and also updated with the operator instructions.